src/explorer/GraphExplorer.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+// A frontend for visualizing Contribution Graphs
+
+import React, {Component} from "react";
+
+type Props = {};
+type State = {};
+
+export class GraphExplorer extends Component<Props, State> {
+  render() {
+    return (
+      <div>
+        <h1> Graph Explorer </h1>
```

Comment:
By universal convention, this should be `<h1>Graph Explorer</h1>`: no spaces inside the angle brackets. (Prettier won't change this because it's technically different HTML output, though it's rendered the same way.)